scalableminds/webknossos
# Display dataset used storage

Show used storage of datasets.

<img src='https://github.com/scalableminds/webknossos/assets/1390035/1c31bb58-f302-42cc-a101-bd703d346061' width='250'>

### Steps to test:
- provoke a storage scan by setting both `fetchUsedStorage.rescanInterval` and `fetchUsedStorage.ticketInterval` to 1 minute, and `reportUsedStorage.enabled` to true
- in dashboard, click on a number of datasets to check displayed size (should be hidden for remote datasets)

### TODOs:
- Backend
  - [x] include used storage in dataset json
- Frontend
  - [x] display the info in the sidebar of the dataset list in dashboard
  - [x] hide values <10KB to avoid confusion

### Issues:
- fixes #7662 

------
